docs/tech-docs/services/computing-resources.md

```diff
@@ -65,3 +65,33 @@ kubectl exec -i -t deployment/reana-server -n reana -- flask reana-admin token-g
 
 6. Navigate to `reana-vre.cern.ch` and log in with your IAM credentials.
 
+## JupyterLab REANA Extension Authentication
+
+This section explains how the VRE-provided JWT used by the REANA JupyterLab extension is injected, accessed and refreshed (or not) during a user session.
+
+### Token injection at spawn time
+During user pod creation the VRE spawner:
+- Obtains (or reuses) the user's already validated access token.
+- Injects it as an environment variable (`REANA_ACCESS_TOKEN`).
+- Writes a lightweight config file with the token for `reana-client` CLI
+
+No refresh token is stored; only the short‑lived access token is passed.
+
+### Token storage (UI extension vs CLI)
+The UI / JupyterLab extension: 
```

Comment:
By UI you mean the Jupyterlba extension, right ? In this case remove UI to avoid misunderstandings, please